dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task230523CreateVariantFieldInContentlet.java

```diff
@@ -0,0 +1,69 @@
+package com.dotmarketing.startup.runonce;
+
+import static com.dotcms.util.CollectionsUtils.list;
+
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import com.dotmarketing.startup.StartupTask;
+import com.liferay.util.StringPool;
+import java.sql.SQLException;
+import java.util.stream.Collectors;
+
+/**
+ * Task to create the variant_id column in the contentlet table, Also it remove the variantId attribute
+ * from the contentlet_as_json column and set the variant_id column to the default value 'DEFAULT'
+ */
```

Comment:
it removes*
sets*

Reply:
done https://github.com/dotCMS/core/pull/25035/commits/4a7b5555a6623425addbb8ea468a2a3201b4d144#diff-8d78df1d2905f3e9ab034c1d88dfdca7ce5d3c7bd13c24b400d1a7116dc1b906R17